reverse_engineering/queries/selectedTablesSubQuery/MemoryOptimizedTablesSubQueryForRetrievingTheTablesSelectedByTheUser.js

```diff
@@ -0,0 +1,34 @@
+const { QueryForRetrievingTheTablesSelectedByTheUser } = require('./QueryForRetrievingTheTablesSelectedByTheUser');
+const { getProjectedPropertiesNames } = require('./getProjectedPropertiesNames');
+
+class MemoryOptimizedTablesSubQueryForRetrievingTheTablesSelectedByTheUser {
```

Comment:
This class can be removed now that we figured out that memory optimized tables are not available in Synapse.